### DISCUSSION TITLE
No Green LED, only 1 CW transmit

### DISCUSSION THREAD

**Discussion by rgrokett** [alanbjohnston/CubeSatSim#231] [Q&A] [answered]
This is just an FYI if anyone runs into similar problem;

I was at section 7.1 Main Board 3 testing. Pi plugged into Main board. Powered up. Got no Green LED but did get a one time Blue CW transmit on 434Mhz, but then no telemetry or other transmission. Shutdown pushbutton worked.  Per docs, the single CW transmit would occur if the Pi isn't plugged into a cubesatsim, but it was. 

Verified Green LED good. Verified the Main Board had all SMD components already soldered in.

Examining code in "rpitx.py" I saw that the Pi determines if it is plugged into a board via GPIO pin 12. Looking at the schematic, I saw that this was just a SMD 1K resistor at R2 to ground the pin. Testing with a meter, the SMD was open. Since the docs showed you can just solder a 1K into R1, I didn't bother more with the SMD resistor.  

Rebooting, I got good Green LED, then Blue lit and CW then good telemetry!  Everything fine. The problem was open R2 SMD with an easy fix (after an hour or so of tracing docs, schematics, and the board). HAPPY FACE! :-) 

Russ WA4EFH

**Comment by rgrokett** [comment, marked as answer]
The problem was open R2 SMD with an easy fix by soldering 1K into R1.

**Reply by alanbjohnston** [reply]
Nice troubleshooting!  I should update the instructions to check R2/R1 if there is no transmit after the CW ID. 

If you could post a photo of the SMD resistor R2. I wonder if the soldering done by the PCB fabricator was bad, or if the resistor was the wrong value or bad.

**Reply by rgrokett** [reply]
Sorry for delay. Attached is the photo of R2 SMD. Looks like it may have slid sideways, but I didn't try just reheating to tell if it was a cold solder or a defective part.  

![R2 SMD](https://user-images.githubusercontent.com/128161/205104454-327cb566-b492-495a-8d3e-f828cbf58f1a.jpg)
